src/middlewared/middlewared/plugins/chart_releases_linux/utils.py

```diff
@@ -62,6 +62,42 @@ def get_chart_release_from_namespace(namespace):
     return namespace.split(CHART_NAMESPACE_PREFIX, 1)[-1]
 
 
+def safe_to_ignore_path(path: str) -> bool:
+    # "/" and "/home/keys/" are added for openebs use only, regular containers can't mount "/" as we have validation
+    # already in place by docker elsewhere to prevent that from happening
+    if path == '/':
+        return True
+
+    for ignore_path in (
+        '/etc/',
+        '/sys/',
+        '/proc/',
+        '/var/lib/kubelet/',
+        '/dev/',
+        '/mnt/',
+        '/home/keys/',
+        '/run/',
+        '/var/run/',
```

Comment:
Ditto for /run and /var/run

Reply:
- Correct: at least `/var/run` should only be needed in containers within their local context, so shouldn't be fully whitelisted.
- I'm going to check how loki and other container-log fetchers behave with these mountpaths, i'll get back to you shortly!

Reply:
Checked, promtail/loki (One of the Apps that requires the biggest host file access of all our Apps) requires:
`/var/log/pods` and `/run/promtail`

The later can be problematic, I think it's best to check weither this `/run/promtail` access thing is a forced requirement though...